tests/unit/importer/test_submission.py

```diff
@@ -181,43 +200,44 @@ def test_submit(self, submission_constructor):
 
         # when:
         submitter = IngestSubmitter(ingest_api)
-        submitter.submit(entity_map, submission_url='url')
+        submitter.add_entities(entity_map, submission_url='url')
 
         # then:
         submission_constructor.assert_called_with(ingest_api, 'url')
         submission.define_manifest.assert_called_with(entity_map)
         submission.add_entity.assert_has_calls([call(product), call(user)], any_order=True)
 
-    @patch('ingest.importer.submission.Submission')
-    def test_submit_update_entities(self, submission_constructor):
+    def test_update_entities(self):
         # given:
         ingest_api = MagicMock('mock_ingest_api')
         ingest_api.get_submission = MagicMock()
-        submission = self._mock_submission(submission_constructor)
+        ingest_api.create_submission_manifest = MagicMock()
+        ingest_api.patch = MagicMock()
 
         # and:
         product = Entity('product', 'product_1', {'k': 'v'})
```

Comment:
I would move this into a function. `createTestData` or similar.

Reply:
Done!